.github/CODEOWNERS

```diff
@@ -22,4 +22,7 @@ opentelemetry/proto/trace/    @open-telemetry/specs-approvers @open-telemetry/sp
 opentelemetry/proto/metrics/  @open-telemetry/specs-approvers @open-telemetry/specs-metrics-approvers
 
 # Logs owners (global + logs)
-opentelemetry/proto/logs/     @open-telemetry/specs-approvers @open-telemetry/specs-logs-approvers
\ No newline at end of file
+opentelemetry/proto/logs/     @open-telemetry/specs-approvers @open-telemetry/specs-logs-approvers
+
+# Profiles owners (global + profiles)
+opentelemetry/proto/profiles  @open-telemetry/specs-approvers @open-telemetry/profiling-maintainers
```

Comment:
Seems that @open-telemetry/profiling-maintainers do not have write access to this repository?

Reply:
I added the permission, should be good now.